### PR TITLE
Template: Add `manifest.contributors` to `nextflow.config`

### PR DESCRIPTION
**Pull request by mirpedrol**
Close https://github.com/nf-core/tools/issues/3289

### CODE REVIEW

File: nf_core/pipeline-template/nextflow.config [hunk 1]
```diff
@@ -274,7 +274,18 @@ dag {
 
 manifest {
     name            = '{{ name }}'
-    author          = """{{ author }}"""
+    author          = """{{ author }}""" // The author field is deprecated from Nextflow version 24.10.0, use contributors instead
+    contributors    = [
+        // TODO nf-core: Update the field with the details of the contributors to your pipeline. New with Nextflow version 24.10.0
+        [
+            name: '{{ author }}',
```
Comment (mashehu) [inline nf_core/pipeline-template/nextflow.config:281]: what happens in case of multiple authors?

Comment (mirpedrol) [reply nf_core/pipeline-template/nextflow.config:281]: you can add another map to the list of maps

Comment (mirpedrol) [reply nf_core/pipeline-template/nextflow.config:281]: ah sorry do you mean the `name` only? should we also make it empty and let people complete?

Comment (mirpedrol) [reply nf_core/pipeline-template/nextflow.config:281]: Then I would leave it empty, because we are allowing both notations for now

Comment (mashehu) [reply nf_core/pipeline-template/nextflow.config:281]: leaving it empty feels weird... why not have a long string in author and then an array for contributor where we split the string?

 we probably should also update the llnting to be able to handle empty author fields.

Comment (mirpedrol) [reply nf_core/pipeline-template/nextflow.config]: I pushed a change using a loop, what do you think?

**Review by mashehu — APPROVED** [review]

File: nf_core/pipeline-template/nextflow.config (hunk 1 above)
Comment (mashehu) [reply nf_core/pipeline-template/nextflow.config]: yes, I think that should be good.

**Pull request merged**